.github/workflows/android.yml

```diff
@@ -23,4 +23,4 @@ jobs:
     - name: Grant execute permission for gradlew
       run: chmod +x gradlew
     - name: Build with Gradle
-      run: ./gradlew build
```

Comment:
don't do this, this is at least partially why the checks are failing, gradlew is the correct name